code-challenges.test.js

```diff
@@ -10,25 +10,62 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
+// a describe method that lists the name of the function OR naming of the particular test.
+// describe("hello", () => {
+//   it("returns a string that says hi", () => {
+//     expect(hello()).toEqual("hi")
+//   })
+// })
+
 // --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 
 // a) Create a test with an expect statement using the variable provided.
 
+// Create a function called theGuide that takes an object and returns each string in that object with capitals applied.
+// Input is an array of objects
+// Output is same array but capitalization is correctly applied to the names
+
+describe("theGuide", () => {
+  it("takes in an array of objects and returns an array with a sentence about each with their name capitalized", () => {
+    expect(theGuide(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
+  })
+})
+
 const people = [
   { name: "ford prefect", occupation: "a hitchhiker" },
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
   { name: "arthur dent", occupation: "a radio employee" }
 ]
+
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
 
 // b) Create the function that makes the test pass.
 
+const theGuide = () =>{
```

Comment:
1. Ensure you are passing the parameter you established in your pseudocode
2. Line 46...You definitely need to split the value which is a string. The split method changed the string into an array.  Therefore, the toUpperCase method will not work. However, if you access the value in the array, then you can use that method. Hint: you may have to store the array first, then on another line perform the steps to do your toUpperCase method.
3. An object in an array can be manipulated by a combination of the index "[]" - to select which object of the array is to be manipulated - and the "." (dot) operator - to apply the operation to the selected object.
4. checkout this site 
https://www.freecodecamp.org/news/how-to-capitalize-words-in-javascript/